artifacts/deployment/etcdproxy-controller.yaml

```diff
@@ -0,0 +1,107 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: kube-apiserver-storage
+spec:
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: etcdproxy-sa
+  namespace: kube-apiserver-storage
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: etcdproxy-crd-cr
+rules:
+- apiGroups: ["etcd.xmudrii.com"]
+  resources: ["etcdstorages"]
+  verbs: ["get", "watch", "list", "create", "update", "patch"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: etcdproxy-res-role
+  namespace: kube-apiserver-storage
+rules:
+- apiGroups: ["apps"]
+  resources: ["replicasets"]
+  verbs: ["get", "watch", "list", "create", "update", "patch"]
+- apiGroups: [""]
+  resources: ["services", "secrets", "configmaps"]
```

Comment:
Is it possible to allow only read (`get`, `watch`, `list`) on ConfigMaps and Secrets related to the core etcd (`etcd-coreserving-ca`, `etcd-coreserving-cert`), but to allow CURD on other ConfigMaps and Secrets (created for each etcdproxy pod)?

What I had in mind is something like:
```
- apiGroups: [""]
  resources: ["secrets", "configmaps"]
  verbs: ["get", "watch", "list", "create", "update", "patch"]
- apiGroups: [""]
  resources: ["configmaps"]
  resourceNames: ["etcd-coreserving-ca"]
  verbs: ["get", "watch", "list"]
- apiGroups: [""]
  resources: ["secrets"]
  resourceNames: ["etcd-coreserving-cert"]
  verbs: ["get", "watch", "list"]
```
But I'm not sure will the first rule override other two rules. Is something like this possible?